laborator/content/buffer-overflow/9-return-adddress/break_this.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+void magic_function(void)
+{
+   system("cowsay -f tux 'Good job there, my friend'");
```

Comment:
```suggestion
   system("cowsay -f tux 'Good job there, my friend'");
```
Indent all the code using tabs (`\t`), not spaces.